.github/workflows/export-dynamic.yaml

```diff
@@ -76,25 +72,15 @@ jobs:
           else
             echo "OVERLAY_REPO_REF=${{ github.head_ref || github.ref_name }}" >> $GITHUB_OUTPUT
           fi
-    
+
       - name: Set environment variables
         id: set-env-vars
         shell: bash
         run: |
-          if [[ "${{  github.event.inputs.node-version }}" != "" ]]
-          then
-            echo "NODE_VERSION=${{ github.event.inputs.node-version }}" >> $GITHUB_OUTPUT
-            echo "UPLOAD_PROJECT_ON_ERROR='${{ github.event.inputs.upload-project-on-error }}'" >> $GITHUB_OUTPUT
-            echo "JANUS_CLI_VERSION=${{ github.event.inputs.janus-cli-version }}" >> $GITHUB_OUTPUT
-          else
-            NODE_VERSION=$(cat node-version 2> /dev/null)
-            NODE_VERSION=${NODE_VERSION:-20.x}
-            echo "NODE_VERSION=$NODE_VERSION" >> $GITHUB_OUTPUT
-            JANUS_CLI_VERSION=$(cat janus-cli-version 2> /dev/null)
-            JANUS_CLI_VERSION=${JANUS_CLI_VERSION:-1.18.0}
-            echo "JANUS_CLI_VERSION=$JANUS_CLI_VERSION" >> $GITHUB_OUTPUT
-            echo "UPLOAD_PROJECT_ON_ERROR='false'" >> $GITHUB_OUTPUT
-          fi
+          echo "NODE_VERSION=${{ github.event.inputs.node-version }}" >> $GITHUB_OUTPUT
```

Comment:
Are these lines even useful ?
I guess we can directly use the `github.event.inputs.xxx` values later on.